CVS-71745 update linux installation guide (#8979)
openvinotoolkit/openvino: docs/install_guides/installing-openvino-windows.md

@endsphinxdirective

## Overview

This guide provides step-by-step instructions on how to install the Intel® Distribution of OpenVINO™ toolkit. Links are provided for each type of compatible hardware including downloads, initialization and configuration steps. The following steps will be covered:

1. <a href="#install-openvino">Install the Intel® Distribution of OpenVINO™ Toolkit</a>
2. <a href="#set-the-environment-variables">Configure the Environment</a>
3. <a href="#model-optimizer">Download additional components (Optional)</a>
4. <a href="#optional-steps">Configure Inference on non-CPU Devices (Optional)</a>  
5. <a href="#get-started">What's next?</a>

## <a name="install-openvino"></a>Step 1: Install the Intel® Distribution of OpenVINO™ toolkit Core Components

1. Download the Intel® Distribution of OpenVINO™ toolkit package file from [Intel® Distribution of OpenVINO™ toolkit for Windows*](https://software.intel.com/en-us/openvino-toolkit/choose-download).
   Select the Intel® Distribution of OpenVINO™ toolkit for Windows* package from the dropdown menu.
   

> 2. Go to the `Downloads` folder and double-click `w_openvino_toolkit_p_<version>.exe`. A window opens to let you choose your installation directory and components. The directory will be referred to as <INSTALL_DIR> elsewhere in the documentation. Once the files extracted, you should see the following dialog box open up:

Reviewer: ```suggestion
2. Go to the `Downloads` folder and double-click `w_openvino_toolkit_p_<version>.exe`. In opened window you can select folder to extract installer files. Once files extracted, you should see the following dialog box:
```

> **NOTE**: If you see an error indicating Python is not installed when you know you installed it, your computer might not be able to find the program. For the instructions to add Python to your system environment variables, see <a href="#Update-Path">Update Your Windows Environment Variables</a>.

The environment variables are set. Next, you can download some additional tools.

## <a name="model-optimizer">Step 3 (Optional): Download additional components

> **NOTE**: Since the OpenVINO™ 2022.1 release, the following development tools: Model Optimizer, Post-Training Optimization Tool, Model Downloader and other Open Model Zoo tools, Accuracy Checker, and Annotation Converter are not part of the installer. These tools are now only available on [pypi.org](https://pypi.org/project/openvino-dev/).

> Among the developer tools you can find e.g. Model Optimizer which imports, converts, and optimizes models, and Model Downloader which gives you access to the collection of pre-trained deep learning public and intel-trained models.  

Reviewer: I believe we should add instruction or reference on how to install  `Install OpenVINO Developer Tools `
* (Recommended) Install manually by using the provided requirements files 
* Install from PyPI repository
Please sync up with  @xu-yuan1 proposal for this (apt & yum install guides)

Author: for now I added placeholder which should be removed once @xu-yuan1 adds the dedicated page
